moml/core/__init__.py

```
@@ -29,30 +29,76 @@
-    extract_fingerprints,
-    # validate_smiles is provided in utils
-)
+try:
+    from .molecular_feature_extraction import (
+        FunctionalGroupDetector,
+        MolecularFeatureExtractor,
+        calculate_molecular_descriptors,
+        extract_fingerprints,
+        # validate_smiles is provided in utils
+    )
+except ImportError as e:
```

Ruff (0.12.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)